File: 268-MissingNumber/268-MissingNumber.py
# Last updated: 7/13/2026, 10:10:38 AM
class Solution:
    def missingNumber(self, nums: List[int]) -> int:
        l=len(nums)
        res=(l*(l+1))/2
        su=sum(nums)
        return int(res-su)
        

File: 268-MissingNumber/268-MissingNumber.java
// Last updated: 7/13/2026, 10:10:32 AM

class Solution {
    public int missingNumber(int[] nums) {
        int n=0,res=0;
        Arrays.sort(nums);
        for(int i:nums){
            if( n==i ) n++;
           
        }
   return n; }
}